File: Conditionals/roots.c
#include<stdio.h>
#include<math.h>

int main(void) {
    int a,b,c,r1,r2;
    float sqrt;
    
    printf("Enter the coefficient of a,b and c, where a/b/c not equal to 0: ");
    scanf("%d %d %d", &a,&b,&c);

    //Roots
    sqrt= (b*b)- (4*a*c);
    r1 = ((-1*b) + sqrtf(sqrt))/(2*a);
    r2 = ((-1*b) - sqrtf(sqrt))/(2*a);

    printf("Coefficient of x^2 is %d, Coefficient of x is %d Y intercept is %d\n",a,b,c);
    printf("The roots of the quadratic is: %d and %d\n", r1,r2);
}

File: CompSci/Tutorials/Tutorial6/roots.py
a,b,c = eval((input("Enter the coefficients of the Equation: ")))

sqrt= (b**2)- (4*a*c)
if sqrt >= 0:
    r1 = ((-1*b) + (sqrt)**(1/2))/(2*a)
    r2 = ((-1*b) - (sqrt)**(1/2))/(2*a)
    if r1 == r2:
        print("This is a double root")
        print("Coefficient of x^2 is",a,"Coefficient of x is",b," and the Y intercept is",c)
        print("The roots of the quadratic are:", r1)
    else:
        print("Coefficient of x^2 is",a,"Coefficient of x is",b," and the Y intercept is",c)
        print("The roots of the quadratic are:", r1,r2)
else:
    print("This equation has no real roots!!!!")


